### PR TITLE
Pets, need pictures of...

### LINKED ISSUE #101: Post pictures of your pets <3
* **Is your feature request related to a problem? Please describe.**

More pics!

* **Describe the solution you'd like**

More pics!

* **Describe alternatives you've considered**

None.

* **Additional context**

This is my dog called Banana.
banana

### PR DESCRIPTION
**Pull request by ran-maciej-idzikowski**
## Proposed changes

Describe the big picture of your changes here to make it clear what your pull request will change. Make sure to explain the **what** e.g. the functionality it changes not the **how** e.g. what code you have written.  If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (change which fixes an issue)
- [x] New feature (change which adds functionality)
- [ ] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [x] I have added unit tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have checked that my changes have not broken any other functionality
- [ ] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [x] All of your unit tests pass
- [ ] Your code is clear and easy to understand
- [ ] You have documented any new features
- [ ] They can run the code you have written
- [ ] Your changes do not break any other functionality

## Who is reviewing this PR?

Tag your reviewer here to ensure they get a notification!

### CODE REVIEW

**Review by TSO-James-Taylor2 — APPROVED** [review]
These are indeed trying times

File: maciejs-file.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+file
+file
+file
+
+no file
```
Comment (JamesTaylor2DfT) [inline maciejs-file.txt:5]: Where did it go?

File: text_file.txt [hunk 2]
```diff
@@ -1 +1,3 @@
-This is a random text file. You can add and remove things to it to test how git works.
\ No newline at end of file
+This is a random text file. You can add and remove things to it to test how git works.
+
+nice
```
Comment (JamesTaylor2DfT) [inline text_file.txt:3]: Nice

**Review by TSO-James-Taylor2 — APPROVED** [review]

**Comment by JamesTaylor2DfT** [comment]
More pets required!

**Comment by ran-maciej-idzikowski** [comment]
All resolved now.

**Pull request closed without merging**